api/views/assets.py

```diff
@@ -86,9 +91,20 @@ def create(self, request, *args, **kwargs):
         return response
 
     def perform_create(self, serializer):
+
+        ManageAssetViewSet.restrict_to_only_super_users(
```

Comment:
Also seeing that aside checking if the user is a superuser it also checks for the presence of an invoice_receipt, making this a static method means this could be used outside here and that is unlikely. You could make the `restrict_to_only_super_users` method an internal method if you want.